Jenkinsfile

```diff
@@ -30,12 +30,6 @@ if (env.BRANCH_NAME.startsWith('release-')) {
   env.BUILD = 'CI'
 }
 
-// Skip sonarQ analysis on branch without PR  - Unable to merge
-def securityAnalysisRequired = 'yes'
-if (!env.CHANGE_ID && env.BUILD == 'CI') {
-    securityAnalysisRequired = 'no'
-}
-
```

Comment:
By removing this part, you will generate a cache tarball on all branches... 
Meaning that you will need to manage cache invalidation